dbhelper.js

```diff
@@ -0,0 +1,20 @@
+const knex = require('knex');
+const dbconfig = require('./knexfile');
+const db = knex(dbconfig.development);
+
+registerUser = (newUser) => {
+    return db('users')
+        .insert(newUser).
```

Comment:
Your helpers are nice. Just try to watch for coding inconsistencies like this. They can be a trouble fixing later when working with a team. the . method should be completely on a new line.